From 1cf676b0394b4062c922510926201f88155ec0a6 Mon Sep 17 00:00:00 2001
From: David Greaves <david@dgreaves.com>
Date: Wed, 6 Nov 2013 21:09:30 +0000
Subject: [PATCH 01/37] (hybris) Add /usr/libexec/droid-hybris/lib-dev-alog/ to
 the LD_LIBRARY_PATH for all init'ed binaries to support the /dev/alog used in
 Mer

Change-Id: If58b8bd7c52b71e6e1f86f714142890dc8c73fd8
---
 rootdir/init.environ.rc.in | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/rootdir/init.environ.rc.in b/rootdir/init.environ.rc.in
index 2e2ab74..deefac4 100644
--- a/rootdir/init.environ.rc.in
+++ b/rootdir/init.environ.rc.in
@@ -1,5 +1,9 @@
 # set up the global environment
 on init
+
+    export PATH /sbin:/vendor/bin:/system/sbin:/system/bin:/system/xbin
+    # This is not 64-bit safe -stskeeps
+    export LD_LIBRARY_PATH /usr/libexec/droid-hybris/lib-dev-alog:/vendor/lib:/system/lib
     export ANDROID_BOOTLOGO 1
     export ANDROID_ROOT /system
     export ANDROID_ASSETS /system/app
-- 
2.7.4

